models/model.py

```diff
@@ -0,0 +1,124 @@
+import pytorch_lightning as pl
+from torch import nn
+from torch import optim
+from torch.optim import AdamW
+from transformers import RobertaModel
+
+
+class ClassifierModule(pl.LightningModule):
+
+    # noinspection PyUnusedLocal
+    def __init__(self, model_hparams, optimizer_hparams):
+        """
+        Inputs:
+            model_hparams - Hyperparameters for the whole model, as dictionary. Also contains Roberta Configuration.
+            optimizer_hparams - Hyperparameters for the optimizer, as dictionary. This includes learning rate,
+            weight decay, etc.
+        """
+        super().__init__()
+
+        # Exports the hyperparameters to a YAML file, and create "self.hparams" namespace
+        self.save_hyperparameters()
+
+        self.loss_module = nn.CrossEntropyLoss()
+
+        self.model = DocumentClassifier(model_hparams)
+
+    def forward(self, inputs):
+        return self.model(inputs)
+
+    def configure_optimizers(self):
+        optimizer = AdamW(self.parameters(), **self.hparams.optimizer_hparams)
+        scheduler = optim.lr_scheduler.StepLR(optimizer, step_size=1,
+                                              gamma=self.hparams.optimizer_hparams['weight_decay'])
+
+        return [optimizer], [scheduler]
+
+    def training_step(self, batch, batch_idx):
+        """
+        Inputs:
+            batch         - Input batch, output of the training loader.
+            batch_idx     - Index of the batch in the dataset (not needed here).
+        """
+
+        # "batch" is the output of the training data loader
+        predictions, labels = self.predict(batch)
+        loss = self.loss_module(predictions, labels)
+
+        self.log('train_accuracy', self.accuracy(predictions, labels).item(), on_step=False, on_epoch=True)
+        self.log('train_loss', loss)
+
+        return loss
+
+    def validation_step(self, batch, batch_idx):
+        # By default logs it per epoch (weighted average over batches)
+        self.log('val_accuracy', self.accuracy(*self.predict(batch)))
+
+    def test_step(self, batch, batch_idx):
+        # By default logs it per epoch (weighted average over batches)
+        self.log('test_accuracy', self.accuracy(*self.predict(batch)))
+
+    @staticmethod
+    def accuracy(predictions, labels):
+        # noinspection PyUnresolvedReferences
```

Comment:
What is this comment referring to?

Reply:
Refers to this: 
![image](https://user-images.githubusercontent.com/8261902/117134133-77707580-ada5-11eb-9ab9-a5de3df173fd.png)

I don't really know why this warning is there in the first place.

Reply:
Ok, it looks like it's something related to Pycharm no?